node/lib/cmd/pull.js

```diff
@@ -78,11 +78,47 @@ used if not specified`,
 };
 
 /**
+ * Return true if the user has requested a rebase (explicitly or
+ * via config).
+ *
+ * @param {Object} args
+ * @param {Boolean} args.rebase
+ * @param {Nodegit.Repository} repo
+ * @param {NodeGit.Branch} branch
+ * @async
+ * @return bool
+ */
+exports.userWantsRebase = co.wrap(function*(args, repo, branch) {
```

Comment:
Can you move this to 'lib/until/pull.js'?  I generally try to keep the code in cmd to a bare minimum and export nothing but the handlers, partially to minimize startup time and also to avoid library code from becoming mingled with argument handlers.